vcell-client/src/main/java/cbit/vcell/client/ClientRequestManager.java

```diff
@@ -2540,6 +2540,9 @@ public static void downloadExportedData(final Component requester, final UserPre
 		if (exportsRecord.contains(evt.getJobID() + "")) {
 			return;
 		}
+		if (evt.getLocation().toLowerCase().endsWith(".n5")){
+			return;
+		}

```

Comment:
maybe make this more robust ... string literal for n5, maybe get info from exportevent?  or case insensitive n5 N5.

Reply:
Changed all string instances of N5 suffix to literal in N5Specs https://github.com/virtualcell/vcell/pull/1047/commits/22e070427f50b17d353fcf2724fc88025bfee054